src/bundle/ui-dev/src/modules/universal-discovery/hooks/useSelectedLocationsHelpers.js

```diff
@@ -0,0 +1,33 @@
+import { useCallback, useContext } from 'react';
+
+import { checkIsSelectable, checkIsSelected, checkIsSelectionBlocked } from '../helpers/selected.locations.helper';
+import {
+    AllowedContentTypesContext,
+    ContainersOnlyContext,
+    ContentTypesMapContext,
+    MultipleConfigContext,
+    SelectedLocationsContext,
+} from '../universal.discovery.module';
+
+export const useSelectedLocationsHelpers = () => {
```

Comment:
I would prefer to remove the word `Helper` from `useSelectedLocationsHelpers` because at the hooks directory level we have a helpers directory with helpers and this can be confusing

Reply:
I think that naming this hook `useSelectedLocations` would be more confusing because with such a name I would expect that the hook would return data about selected locations. But instead, it returns helpers so I think that `useSelectedLocationsHelpers` precisely says what we have here and removing `Helpers` from it would make it more confusing.